pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -394,7 +394,11 @@ func (o *LoginOptions) SaveConfig() (bool, error) {
 	}
 
 	if err := kclientcmd.ModifyConfig(o.PathOptions, *configToWrite, true); err != nil {
```

Comment:
@liggitt hm, I could parse the error string to make sure it is a "permission denied" error before returning this message, and then otherwise return just the original error message.